api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -765,6 +765,28 @@ message HttpConnectionManager {
   // If this config is set, the Proxy-Status HTTP response header field is
   // populated. By default, it is not.
   ProxyStatusConfig proxy_status_config = 49;
+
+  // Configuration options for Header Validation (UHV).
+  // UHV is an extensible mechanism for checking validity of HTTP requests as well as providing
+  // normalization for request attributes, such as URI path.
+  // If the typed_header_validation_config is present it overrides the following options:
+  // `normalize_path`, `merge_slashes`, `path_with_escaped_slashes_action`
+  // `http_protocol_options.allow_chunked_length`.
+  //
+  // The default UHV checks the following:
+  // 1. HTTP/1 header map validity according to `RFC 7230 section 3.2<https://tools.ietf.org/html/rfc7230#section-3.2>`_
+  // 2. Syntax of HTTP/1 request target URI and response status
+  // 3. HTTP/2 header map validity according to `RFC 7540 section 8.1.2<https://tools.ietf.org/html/rfc7540#section-8.1.2`_
+  // 4. Syntax of HTTP/2 pseudo headers
+  // 5. HTTP/3 header map validity (TBD standard link when approved)
```

Comment:
wait is there not an HTTP/3 RFC?  @RyanTheOptimist @DavidSchinazi

Reply:
It's *very* close to publication but [not quite there yet](https://www.rfc-editor.org/current_queue.php#draft-ietf-quic-http). It will be RFC 9114 once published.

Reply:
Oy, I naively assumed it landed with 9000.  Thanks for the update :-)
I think it's worth a link to the draft with a WIP qualifier but your call Yan.

Reply:
Done

Reply:
@DavidSchinazi, @alyssawilk 9114 is published! https://www.rfc-editor.org/rfc/rfc9114.html